src/world/entity/mod.rs

```diff
@@ -73,6 +79,93 @@ pub struct Common {
+            table.insert("DANCER", "¹«Èñ");
+            table.insert("NOVICE_H", "ÃÊº¸ÀÚ");
+            table.insert("SWORDMAN_H", "°Ë»Ç");
+            table.insert("MAGICIAN_H", "À§Àúµå");
+            table.insert("ARCHER_H", "±Ã¼Ö");
+            table.insert("ACOLYTE_H", "¼ºÁ÷ÀÚ");
+            table.insert("MERCHANT_H", "»ÓÀÎ");
+            table.insert("THIEF_H", "ΜΜΜÏ");
+            table.insert("KNIGHT_H", "·Îµå³ªÀÌÆ®");
+            table.insert("PRIEST_H", "ÇÏÀÌÇÁ¸®");
+            table.insert("WIZARD_H", "ÇÏÀÌÀ§Àúµå");
+            table.insert("BLACKSMITH_H", "È­ÀÌÆ®½º¹Ì½º");
+            table.insert("HUNTER_H", "½º³ªÀÌÆÛ");
+            table.insert("ASSASSIN_H", "¾î½Ø½ÅÅ©·Î½º");
+            table.insert("CRUSADER_H", "ÆÈ¶óµò");
+            table.insert("MONK_H", "Ã¨ÇÇ¿Â");
+            table.insert("SAGE_H", "ÇÁ·ÎÆä¼­");
+            table.insert("ROGUE_H", "½ºÅäÄ¿");
+            table.insert("ALCHEMIST_H", "Å©¸®¿¡ÀÌÅÍ");
+            table.insert("BARD_H", "Å¬¶ó¿î");
+            table.insert("DANCER_H", "Áý½Ã");
+            table.insert("RUNE_KNIGHT", "·é³ªÀÌÆ®");
+            table.insert("WARLOCK", "¿ö·Ï");
+            table.insert("RANGER", "·¹ÀÎÁ®");
+            table.insert("ARCH_BISHOP", "¾ÆÅ©ºñ¼ó");
+            table.insert("MECHANIC", "¹ÌÄÉ´Ð");
+            table.insert("GUILLOTINE_CROSS", "±æ·ÎÆ¾Å©·Î½º");
+            table.insert("ROYAL_GUARD", "°¡ΜÅ");
+            table.insert("SORCERER", "¼Ò¼­·¯");
+            table.insert("MINSTREL", "¹Î½ºÆ®·²");
+            table.insert("WANDERER", "¿ø´õ·¯");
+            table.insert("SURA", "½´¶ó");
+            table.insert("GENETIC", "Á¦³×¸¯");
+            table.insert("SHADOW_CHASER", "½¦µµ¿ìÃ¼ÀÌ¼­");
+            table.insert("RUNE_KNIGHT_H", "·é³ªÀÌÆ®");
+            table.insert("WARLOCK_H", "¿ö·Ï");
+            table.insert("RANGER_H", "·¹ÀÎÁ®");
+            table.insert("ARCH_BISHOP_H", "¾ÆÅ©ºñ¼ó");
+            table.insert("MECHANIC_H", "¹ÌÄÉ´Ð");
+            table.insert("GUILLOTINE_CROSS_H", "±æ·ÎÆ¾Å©·Î½º");
+            table.insert("ROYAL_GUARD_H", "°¡ΜÅ");
+            table.insert("SORCERER_H", "¼Ò¼­·¯");
+            table.insert("MINSTREL_H", "¹Î½ºÆ®·²");
+            table.insert("WANDERER_H", "¿ø´õ·¯");
+            table.insert("SURA_H", "½´¶ó");
+            table.insert("GENETIC_H", "Á¦³×¸¯");
+            table.insert("SHADOW_CHASER_H", "½¦µµ¿ìÃ¼ÀÌ¼­");
+            table.insert("SUPERNOVICE", "½´ÆÛ³ëºñ½º");
+            table.insert("GUNSLINGER", "°Ç³Ê");
+            table.insert("NINJA", "´ÑÀÚ");
+            table.insert("TAEKWON", "ÅÂ±Ç¼Ò³â");
+            table.insert("STAR", "±Ç¼º");
+            table.insert("LINKER", "¼Ò¿ï¸µÄ¿");
+            table.insert("SUPERNOVICE2", "½´ÆÛ³ëºñ½º");
+            table.insert("KAGEROU", "KAGEROU");
+            table.insert("OBORO", "OBORO");
+
+            table
+        };
+    }
+
+    JOB_SPRITE_NAME_TABLE.get(player_job).copied().unwrap_or("ÃÊº¸ÀÚ")
+}
+
```

Comment:
If we do hard code this for now, I think it would be better to use a `match` instead of creating a `HashMap` and indexing it. So basically this code would become
```rust
match player_job {
    "NOVICE" => "ÃÊº¸ÀÚ",
    "SWORDMAN" => "°Ë»Ç",
    /* ... */
    _ => "ÃÊº¸ÀÚ",
}
```
Then again, it is possible that an index into a `HashMap` is faster than the `switch` statement and if you have good reason to believe that then I can be convinced to keep the hash map. Maybe you could use the new built-in profiler to test the performance of both solutions? :eyes:

Reply:
> If we do hard code this for now, I think it would be better to use a `match` instead of creating a `HashMap` and indexing it. So basically this code would become
> 
> ```rust
> match player_job {
>     "NOVICE" => "ÃÊº¸ÀÚ",
>     "SWORDMAN" => "°Ë»Ç",
>     /* ... */
>     _ => "ÃÊº¸ÀÚ",
> }
> ```
> 
> Then again, it is possible that an index into a `HashMap` is faster than the `switch` statement and if you have good reason to believe that then I can be convinced to keep the hash map. Maybe you could use the new built-in profiler to test the performance of both solutions? 👀

Using switch with job id should be better